### ISSUE TITLE
No property 'schema_id' in property list (or its value was undefined)

### ISSUE THREAD

**Issue by jonnius** [sustmi/gnome-shell-extension-workspace-switcher-popup-thumbnails#6]
Does not work for me on Ubuntu 18.10 with GNOME Shell 3.28.3. Neither the thumbnail, nor the default animation is shown. This is the error output:
```
(gnome-shell:5953): Gjs-WARNING **: 19:58:15.329: JS ERROR: Error: No property 'schema_id' in property list (or its value was undefined)
_init@/home/me/.local/share/gnome-shell/extensions/workspace-switcher-popup-thumbnails@sustmidown.centrum.cz/thumbnailsBox.js:99:26
wrapper@resource:///org/gnome/gjs/modules/_legacy.js:82:22
_Base.prototype._construct@resource:///org/gnome/gjs/modules/_legacy.js:18:5
Class.prototype._construct/newClass@resource:///org/gnome/gjs/modules/_legacy.js:114:32
enable/workspaceSwitcherPopupInjections._init<@/home/me/.local/share/gnome-shell/extensions/workspace-switcher-popup-thumbnails@sustmidown.centrum.cz/extension.js:53:29
replaceFunction/parent[name]@/home/me/.local/share/gnome-shell/extensions/workspace-switcher-popup-thumbnails@sustmidown.centrum.cz/monkeyPatch.js:46:16
_Base.prototype._construct@resource:///org/gnome/gjs/modules/_legacy.js:18:5
Class.prototype._construct/newClass@resource:///org/gnome/gjs/modules/_legacy.js:114:32
_showWorkspaceSwitcher@resource:///org/gnome/shell/ui/windowManager.js:1939:48
wrapper@resource:///org/gnome/gjs/modules/_legacy.js:82:22
```

**Comment by sustmi** [comment]
Hi @jonnius .
Which version do you use? From the stack trace I guess that it is `v1` (eae1162). I believe that the issue should be fixed in the most recent code (a66d55f). However this version (`v2`) is still under a review on extensions.gnome.org.
Can you verify that the not yet released version still has this issue?

**Comment by jonnius** [comment]
I can confirm that it is fixed with the latest version. Thanks!
